ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ComplianceClusterStatus.tsx

```diff
@@ -35,6 +35,16 @@ function ComplianceClusterStatus({ errors }: ComplianceClusterStatusProps) {
               };
     }
 
+    function getErrorsList(errors: string[]): ReactElement {
+        return (
+            <SimpleList>
+                {errors.map((error) => {
+                    return <SimpleListItem key={error}>{error}</SimpleListItem>;
```

Comment:
@mtodor Can we update the message to something like this:

```
The "compliance-operator" deployment was not found in any namespace. The installed version "v1.5.1" is unsupported; the minimum required version is "v1.6.0".
```

(cc: @mansursyed @zhenpesky )